dotnet/src/dotnetframework/GxClasses/Core/GXUtilsCommon.cs

```diff
@@ -5953,6 +6060,11 @@ public static string Save(Image bitmap, string imageFile, ImageFormat format)
 				ms.Position = 0;
 				try
 				{
+					if (imageFile.StartsWith("http://") || imageFile.StartsWith("https://"))
```

Comment:
If possible, I suggest replacing the hard-coded string with constants:

					Uri uri;
					if (Uri.TryCreate(imageFile, UriKind.Absolute, out uri) && (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps))
					{ 
						imageFile = Path.GetFileName(uri.AbsolutePath);
					}